packages/bpk-component-spinner/src/BpkExtraLargeSpinner.js

```diff
@@ -26,18 +28,26 @@ import SPINNER_TYPES from './spinnerTypes';
 
 const getClassName = cssModules(STYLES);
 
-const BpkExtraLargeSpinner = props => {
+type Props = {
+  type: string,
```

Comment:
This is too broad, we should use `$Keys<typeof SPINNER_TYPES>`. As a rule of thumb, Flow types should match what we put in propTypes

Reply:
Did this suggestion and seems to work fine